recipes/r-modeltime.resample/meta.yaml

```diff
@@ -0,0 +1,124 @@
+{% set version = "0.2.3" %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-modeltime.resample
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/modeltime.resample_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/modeltime.resample/modeltime.resample_{{ version }}.tar.gz
+  sha256: 7b78fba8338cf01f7f70ea4e254aac96ff0a3d1b04ab45bb9d9047a30b262c5f
+
+build:
+  merge_build_host: true  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-cli
+    - r-dials
+    - r-doparallel
+    - r-dplyr
+    - r-forcats
+    - r-foreach
+    - r-forecast
+    - r-ggplot2
+    - r-glue
+    - r-gt
+    - r-hardhat >=1.0.0
+    - r-janitor
+    - r-magrittr
+    - r-parallelly
+    - r-parsnip >=0.2.1
+    - r-plotly
+    - r-prophet
+    - r-purrr
+    - r-reactable
+    - r-rlang >=0.1.2
+    - r-scales
+    - r-stanheaders
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-tidymodels
+    - r-tidyverse
+    - r-timetk >=2.8.1
+    - r-workflows >=1.0.0
+    - r-xgboost >=1.2.0.1
+    - r-yardstick >=0.0.8
+    - r-modeltime >=0.3.0
+    - r-rsample
+    - r-recipes
+    - r-crayon
+    - r-progressr
+    - r-tictoc
+    - r-tune
+  run:
+    - r-base
+    - r-cli
+    - r-dials
+    - r-doparallel
+    - r-dplyr
+    - r-forcats
+    - r-foreach
+    - r-forecast
+    - r-ggplot2
+    - r-glue
+    - r-gt
+    - r-hardhat >=1.0.0
+    - r-janitor
+    - r-magrittr
+    - r-parallelly
+    - r-parsnip >=0.2.1
+    - r-plotly
+    - r-prophet
+    - r-purrr
+    - r-reactable
+    - r-rlang >=0.1.2
+    - r-scales
+    - r-stanheaders
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-tidymodels
+    - r-tidyverse
+    - r-timetk >=2.8.1
+    - r-workflows >=1.0.0
+    - r-xgboost >=1.2.0.1
+    - r-yardstick >=0.0.8
+    - r-modeltime >=0.3.0
+    - r-rsample
+    - r-recipes
+    - r-crayon
+    - r-progressr
+    - r-tictoc
+    - r-tune
+
+test:
+  commands:
+    - $R -e "library('modeltime.resample')"           # [not win]
+    - "\"%R%\" -e \"library('modeltime.resample')\""  # [win]
+
+about:
+  home: https://github.com/business-science/modeltime.resample
+  license: MIT
+  summary: "A 'modeltime' extension that implements forecast resampling tools that assess time-based model performance and stability for a single time series, panel data, and cross-sectional time series analysis."
+  license_family: MIT
+  license_file:
+    - {{ environ["PREFIX"] }}/lib/R/share/licenses/MIT
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
```

Comment:
Did you want to be a maintainer on the feedstock? If so, add your GitHub user name to the list. This is not a requirement - just wondering.